blak05-cv09

krystofblak05 opened the pull request:
https://eso.vse.cz/~blak05/cv09/index.php

krystofblak05 committed:
blak05-cv09

https://eso.vse.cz/~blak05/cv09/index.php

krystofblak05 commented:
Vidím, že kolegové sem přidali i údaje pro admin účet - mail: admin@vse.cz, heslo: admin

nvbach91 commented on www/blak05/cv09/auth2.php:
```diff
@@ -0,0 +1,9 @@
+<?php
+
+$number = @$_COOKIE['privilege'];
+echo $number;
```
zakomentovat

nvbach91 commented on www/blak05/cv09/users.php:
```diff
@@ -0,0 +1,56 @@
+        <label class="form-label">Email:</label>
+        <input name="email" class="form-control" type="email" value="<?php echo $user["email"]?>">
+        <label class="form-label">Privilege:</label>
+        <input name="privilege" class="form-control" type="privilege" value="<?php echo $user['privilege']?>">
```
chtelo by to vytvorit select - z jakych moznych opravneni lze vybrat, nenechat jako volny text

krystofblak05 committed:
blak05-cv09

V Users nahrazen input selectem s volbou privilegií 1 nebo 2. V Auth2 zakomentováno echo.
